MFVNeutralino/test/ntuple.py

```diff
@@ -10,39 +10,65 @@
 settings.run_n_tk_seeds = False
 settings.minitree_only = False
 settings.prepare_vis = False
-settings.keep_all = False
+settings.keep_all = False #FIXME 
 settings.keep_gen = False
 settings.keep_tk = False
 if use_btag_triggers :
     #settings.event_filter = 'dilepton only' # for new trigger studies
     #settings.event_filter = 'leptons only' # for new trigger studies
-    settings.event_filter = 'low HT online track test' # for new trigger studies
-    #settings.event_filter = 'bjets OR displaced dijet' # for new trigger studies
+    #settings.event_filter = 'low HT online track test' # for new trigger studies
+    settings.event_filter = 'bjets OR displaced dijet' # for new trigger studies
 elif use_MET_triggers :
     settings.event_filter = 'met only'
+elif use_Muon_triggers :
+    settings.event_filter = 'muons only' #FIXME
+elif use_Electron_triggers :
+    settings.event_filter = 'electrons only' #FIXME
```

Comment:
What are the FIXMEs for? I generally suggest we drop as much commented-out things from this file as possible, to keep it simple (e.g. there are a bunch of places with different input files or custom sample choices in use_Muon_triggers). Mostly so we don't accidentally miss an important configuration option that we forgot we enabled.